usagov/usagov-2021
# USAGOV-448-Require-Page-Type

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-448

## Description

Makes pagetype a required field and set the default to "Standard Page"

## How to test
- Content -> Add Content -> Basic Page
- Find the `Page Type` field
- Confirm that it is required and there is no option to set it to `None`

## Checklist for the Developer



## Checklist for the Peer Reviewers
